ci/docker/tizen/tools/cache-checksum.sh

```diff
@@ -40,7 +40,7 @@ if [[ "$COMMAND" == "restore" ]]; then
     fi
 
     # Set mtime of files in $TARGET_DIRS to OLD time.
-    for d in $(find $TARGET_DIRS -type d -not -path "*/.git*"); do
+    for d in $(find $TARGET_DIRS -type d); do
```

Comment:
If mtime is temporary set to OLD time for all .git files, then shouldn't we also do validation with checksum?

Reply:
From the perspective of the Incremental build, there is no need to forward the .git files to ninja's changed files input.